apps/ptg-angular-app/src/app/dashboard/tooltip-component/tooltip.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ptg-ui-dialog-component',
```

Comment:
@poojacsawai Here is the PR of Tooltip component and you are writing `ptg-ui-dialog-component`